mailpoet/lib/Form/Block/BlockRendererHelper.php

```diff
@@ -34,7 +34,7 @@ public function getInputValidation(array $block, array $extraRules = [], ?int $f
       $rules['required'] = true;
       $rules['minlength'] = ModelValidator::EMAIL_MIN_LENGTH;
       $rules['maxlength'] = ModelValidator::EMAIL_MAX_LENGTH;
-      $rules['error-message'] = __('Please specify a valid email address.', 'mailpoet');
+      $rules['type-message'] = __('This value should be a valid email', 'mailpoet');
```

Comment:
I'm not sure about this message. The change makes sense to me, but the text has a slightly different stylization than other messages. But maybe we should address all messages and UX with error messages in a new task. I'm only curious about your opinion.

Reply:
That's a good point about the stylization. On the other hand, I like the new message more :) I don't have a strong opinion here.

Reply:
Okay. Let's leave it as it is. Thanks